apps/swirl-docs/src/scripts/prebuild.ts

```diff
@@ -1,22 +1,21 @@
-import { isProdDeployment } from "@swirl/lib/env";
+import { ApiEndpointGenerator } from "./DeploymentStrategy/ApiEndpointGenerator";
+import { FileFetcher } from "./DeploymentStrategy/FileFetcher";
 import {
-  DeploymentStrategy,
-  ProductionDeployment,
-  StagingDeployment,
-} from "./DeploymentStrategy";
+  ApiDocsNavigationGenerator,
+  ApiSpecsNavigationGenerator,
+} from "./DeploymentStrategy/NavigationGenerator";
```

Comment:
i think the deployment strategy is now the wrong name. there once was the idea to only generate stuff for "prod" and for "staging". this is no longer the case so i think we can just go over the "Strategy" and call it generators directly